### DISCUSSION TITLE
academy/api

### DISCUSSION THREAD

**Discussion by giscus** [apify/apify-docs#595] [Comments]
# academy/api

A collection of various tutorials explaining how to interact with the Apify platform programmatically using its API.

https://docs.apify.com/academy/api

**Comment by JulieWang0** [comment]
The above link for "Python" still direct me to "Apify API client for JavaScript". Could you please take a look? Thanks!

**Reply by B4nan** [reply]
Thanks for the report, we are apparently having some broken redirects there. The python docs are here (note the trailing slash):

https://docs.apify.com/api/client/python/
